packages/back-end/src/controllers/organizations.ts

```diff
@@ -454,6 +455,59 @@ export async function putMemberRole(
   }
 }
 
+export async function putInviteRole(
+  req: AuthRequest<{ role: MemberRole }, { key: string }>,
+  res: Response
+) {
+  req.checkPermissions("organizationSettings");
+
+  const { org } = getOrgFromReq(req);
+  const { role } = req.body;
+  const { key } = req.params;
+  const originalInvites: Invite[] = org.invites;
```

Comment:
This won't work since it's doing a shallow copy.  Below when you do `m.role = role`, it will also change this originalInvites variable.

Instead, you can do a deep clone.  Easiest solution is:
```ts
import cloneDeep from "lodash/cloneDeep";
...

const originalInvites = cloneDeep(org.invites);
```